lib/smart_answer_flows/pay-leave-for-parents/outcomes/outcome_mat_leave_mat_pay.erb

```diff
@@ -3,9 +3,9 @@
 
   <%= render partial: 'mat_pay', locals: { calculator: calculator } %>
 
-  <%= render partial: 'extra_help' %>
-
   <% if calculator.two_carers? %>
     <%= render partial: 'paternity_unavailable' %>
   <% end %>
+
+  <%= render partial: 'extra_help' %>
```

Comment:
Could this commit be fixed up to the commit where this block is added?

```
  <% if calculator.two_carers? %>
    <%= render partial: 'paternity_unavailable' %>
  <% end %>
```

No point committing code in one commit and then moving it in the other 👍

Reply:
I see what you mean...this is a symptom of changes being made months apart I think